src/quacc/recipes/espresso/core.py

```diff
@@ -80,6 +81,7 @@ def relax_job(
     relax_cell: bool = False,
     parallel_info: dict[str] | None = None,
     copy_files: str | Path | list[str | Path] | None = None,
+    test_run: bool = False,
```

Comment:
Please add a docstring for `test_run`  in both places added. Also, while you're at it, if you can have it be above `copy_files` that'd be great (I usually reserve that one for last, before the **kwargs). Just make sure the docstring placement matches.

Reply:
Done!

Reply:
Great! There's a failing Espresso test, btw.

Reply:
Yes, and quite important: the test_run part wasn't working! It should be better now